src/utils/auth_utils.jsx

```diff
@@ -0,0 +1,420 @@
+/* eslint-disable no-console */
+/* eslint-disable react/react-in-jsx-scope */
+import axios from 'axios';
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
🚫 **[eslint]** <[import/no-extraneous-dependencies](https://github.com/import-js/eslint-plugin-import/blob/v2.25.4/docs/rules/no-extraneous-dependencies.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'uuid' should be listed in the project's dependencies. Run 'npm i -S uuid' to add it